Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -54,3 +62,61 @@
     display: flex;
 }
 
+.card {
+    padding: 0 12px 29px 5px;
+    border-width: 2px;
+    border-color: var(--color-card-border);
+    flex-basis: 42%;
+    margin-right: 24px;
+}
+
+.card {
+    background-color: var(--color-card-background);
+}
+
+.card--small {
+    padding: 0;
+    margin: 0 0 13px 6px;
+    flex-direction: row;
+    flex-basis: auto;
+}
+
+.card__icon--small {
+    flex-grow: 1;
+    background-color: rgba(0, 0, 0, 0);
+    width: 50px;
+    height: 50px;
+    scale: 0.5 0.5;
+    align-self: stretch;
+    align-items: stretch;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    flex-direction: row;
+    -unity-text-align: upper-left;
+    bottom: 5px;
+    max-width: none;
+    min-width: 50px;
+    left: auto;
+    right: 9px;
+    min-height: 50px;
+}
+
+.card__text--small {
```

Comment:
As pointed out in a previous CR, we should establish a convention for the naming. Could you clarify when you are choosing to use `__` vs using Pascal casing. I understand the `--` notation for identifiers, which seems fine.

Reply:
I think in the initial PR, we should move to using lowercase then __ just to keep consistency with the refactored code.